app/src/ui/app.tsx

```diff
@@ -653,6 +654,33 @@ export class App extends React.Component<IAppProps, IAppState> {
     }
   }
 
+  private discardChanges = (
+    repository: Repository,
+    files: ReadonlyArray<WorkingDirectoryFileChange>
+  ) => {
+    if (!repository) {
```

Comment:
This check should be unnecessary, right?

Reply:
✅